src/coreclr/vm/interpexec.cpp

```diff
@@ -291,6 +293,67 @@ template <typename TResult, typename TSource> void ConvOvfHelper(int8_t *stack,
+                    MethodTable *pMT = nullptr;                                                             \
+                    MethodDesc* pMD = nullptr;                                                              \
+                    uint8_t* lookup;                                                                        \
+                    if (pLookup->lookupType == InterpGenericLookupType::This)                               \
+                    {                                                                                       \
+                        OBJECTREF thisPtr = LOCAL_VAR(genericVar, OBJECTREF);                               \
+                        NULL_CHECK(thisPtr);                                                                \
+                        pMT = thisPtr->GetMethodTable();                                                    \
+                        lookup = (uint8_t*)pMT;                                                             \
+                    }                                                                                       \
+                    else if (pLookup->lookupType == InterpGenericLookupType::Class)                         \
+                    {                                                                                       \
+                        pMT = LOCAL_VAR(genericVar, MethodTable*);                                          \
+                        lookup = (uint8_t*)pMT;                                                             \
+                    }                                                                                       \
+                    else                                                                                    \
+                    {                                                                                       \
+                        assert(pLookup->lookupType == InterpGenericLookupType::Method);                     \
+                        pMD = LOCAL_VAR(genericVar, MethodDesc*);                                           \
+                        lookup = (uint8_t*)pMD;                                                             \
+                    }                                                                                       \
+                    void* result = 0;                                                                       \
+                                                                                                            \
+                    if (pLookup->indirections != 0)                                                         \
+                    {                                                                                       \
+                        lookup = *(uint8_t**)(lookup + pLookup->offsets[0]);                                \
+                        if (pLookup->indirections >= 3)                                                     \
```

Comment:
This and the 4 look like they're off by 1, but I assume that's on purpose and goes with the 'last indirection is the size' below